src/material/dialog/dialog-container.ts

```diff
@@ -145,7 +145,7 @@ function parseCssTime(time: string | number | undefined): number | null {
     '[attr.aria-modal]': '_config.ariaModal',
     '[id]': '_config.id',
     '[attr.role]': '_config.role',
-    '[attr.aria-labelledby]': '_config.ariaLabel ? null : _ariaLabelledBy',
+    '[attr.aria-labelledby]': '_config.ariaLabel ? null : _ariaLabelledByQueue[0]',
```

Comment:
Note that `aria-labelledby` supports passing a list of IDs so we could've also used all the IDs here, but I think that for the dialog case it makes sense to only have one label for the entire dialog.